libcaf_net/caf/net/web_socket/framing.cpp

```diff
@@ -212,6 +251,10 @@ ptrdiff_t framing::handle(uint8_t opcode, byte_span payload,
   // opcodes are checked for validity when decoding the header
   switch (opcode) {
     case detail::rfc6455::connection_close:
+      if (auto error = validate_closing_payload(payload); error) {
+        abort_and_shutdown(error);
```

Comment:
We generally name this `err`, it's also good to avoid `error` because it introduces ambiguity with `caf::error`.

```suggestion
      if (auto err = validate_closing_payload(payload); err) {
        abort_and_shutdown(err);
```